articles/data-science-setup/index.md

```diff
@@ -0,0 +1,119 @@
+### Introduction
+
+The path to becoming a data scientist begins with setting up your computer with the appropriate tools. The majority of the data science ecosystem is built around Python and R. In this article, we will create a basic Python setup using the Anaconda Package Manager. This guide will guide you to install popular machine learning packages.   
+
+### Table of contents
```

Comment:
Can you add links to the table of contents such that they link within the article? And some headings are missing from the table of contents.